.github/CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 <h3>New features since last release</h3>
 
+* The quantum state of a QNode can now be returned using the ``state()`` return function.
+  [(#818)](https://github.com/XanaduAI/pennylane/pull/818)
+  
+  Consider the following QNode:
+  ```python
+  import pennylane as qml
+  from pennylane.beta.tapes import qnode
+  from pennylane.beta.queuing import state
+  
+  dev = qml.device("default.qubit", wires=3)
+  
+  @qnode(dev)
+  def qfunc(x, y):
+      qml.RZ(x, wires=0)
+      qml.CNOT(wires=[0, 1])
+      qml.RY(y, wires=1)
+      qml.CNOT(wires=[0, 2])
+      return state()
```

Comment:
Would `state` take a `wires` argument? :thinking:

Reply:
Ah I knew I'd forget somewhere :laughing: Yes, it now shouldn't take a wires argument after an update based on feedback.

Reply:
(description updated)

Reply:
Oh awesome! It was me, was coming as a late reviewer 😊